gulpfile.js

```diff
@@ -15,28 +15,46 @@ var phpwatcher = require('./lib/phpwatcher');
 var less = require('./lib/less');
 var lesswatcher = require('./lib/lesswatcher');
 
+var knownOptions = {
+  boolean: 'no.symlinks',
```

Comment:
If you use `symlinks` here minimist automatically parses `--no-symlinks` and `--symlinks`. Then code can be written without the double-negative: `if (options.symlinks) { ... }`